src/PowerShellEditorServices/Services/Workspace/LanguageServerSettings.cs

```diff
@@ -353,6 +357,27 @@ public class CodeFoldingSettings
         }
     }
 
+    /// <summary>
+    /// Pester settings
+    /// </summary>
+    public class PesterSettings
+    {
+        public PesterSettings()
+        {
+
+        }
+
+        public PesterSettings(PesterSettings settings)
+        {
+            Pester5CodeLens = settings.Pester5CodeLens;
+        }
+
+        /// <summary>
+        /// Whether integration features specific to Pester v5 are enabled
+        /// </summary>
+        public bool Pester5CodeLens { get; set; } = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 'Pester5CodeLens', the compiler will do that for you.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4894668)